src/algokit/cli/tasks/send_transaction.py

```diff
@@ -0,0 +1,166 @@
+
+    Raises:
+        click.ClickException: If the piped transaction content is invalid.
+    """
+    # Read the raw file content from the standard input
+
+    raw_file_content = cast(TextIOWrapper, click.get_text_stream("stdin")).read()
+
+    try:
+        # Parse the raw file content as JSON
+        file_content = json.loads(raw_file_content)
+    except json.JSONDecodeError as ex:
+        raise click.ClickException("Invalid piped transaction content!") from ex
+
+    # Check if the content is a list of dicts with the required fields
+    if not isinstance(file_content, list) or not all(_is_sign_task_output_txn(item) for item in file_content):
+        raise click.ClickException("Invalid piped transaction content!")
+
+    # Convert the content into SignedTransaction objects
+    return [encoding.msgpack_decode(item["content"]) for item in file_content]  # type: ignore[no-untyped-call]
+
+
+def _get_signed_transactions(file: Path | None = None, transaction: str | None = None) -> list[SignedTransaction]:
+    """
+    Retrieves a list of signed transactions.
+
+    Args:
+        file (Optional[Path]): A `Path` object representing the file path from which to retrieve the transactions.
+        transaction (Optional[str]): A base64 encoded string representing a single signed transaction.
+
+    Returns:
+        list[SignedTransaction]: A list of `SignedTransaction` objects representing the retrieved signed transactions.
+
+    Raises:
+        click.ClickException: If the supplied transaction is not of type `SignedTransaction`.
+        click.ClickException: If there is an error decoding the transaction.
+
+    """
+    try:
+        if file:
+            txns = retrieve_from_file(str(file))  # type: ignore[no-untyped-call]
+        elif transaction:
+            txns = [encoding.msgpack_decode(transaction)]  # type: ignore[no-untyped-call]
+        else:
+            txns = _load_from_stdin()
+
+        for txn in txns:
+            if not isinstance(txn, SignedTransaction):
+                raise click.ClickException("Supplied transaction is not signed!")
+
+        return cast(list[SignedTransaction], txns)
+
+    except Exception as ex:
+        logger.debug(ex, exc_info=True)
+        raise click.ClickException(
+            "Failed to decode transaction! If you are intending to send multiple transactions use `--file` instead."
+        ) from ex
+
+
+def _send_transactions(network: str, txns: list[SignedTransaction]) -> None:
+    """
+    Sends a list of signed transactions to the Algorand blockchain network using the AlgodClient.
+
+    Args:
+        network (str): The network to which the transactions will be sent.
+        txns (list[SignedTransaction]): A list of signed transactions to be sent.
+
+    Returns:
+        None: The function does not return any value.
+    """
+    algod_client = load_algod_client(network)
+
+    if any(txn.transaction.group for txn in txns):
```

Comment:
If the transaction group doesn't match on all of the transactions, what would happen?

Reply:
@robdmoore algosdk will handle that and throw `inconsistent group values` error during sending the transaction group